src/common/snippets/include/snippets/lowered/pass/brgemm_blocking.hpp

```diff
@@ -112,9 +113,68 @@ class BrgemmBlocking : public snippets::lowered::pass::RangedPass, public Brgemm
             size_t m_block, n_block, k_block;
             std::tie(m_block, n_block, k_block) = get_blocking_params(brgemm_expr);
             modified = mark_blocking_loops(linear_ir, expr_it, m_block, n_block, k_block);
+            SNIPPETS_DEBUG_PARAMS_DUMP(brgemm_expr, this);
         }
         return modified;
     }
+
+#ifdef SNIPPETS_DEBUG_CAPS
+    std::string params_to_csv(const ov::snippets::lowered::ExpressionPtr& brgemm_expr) {
+        const auto brgemm = ov::as_type_ptr<BRGEMM_TYPE>(brgemm_expr->get_node());
+        OPENVINO_ASSERT(brgemm, "Brgemm is nullptr!");
+        std::stringstream ss;
+        for (size_t i = 0; i < brgemm->get_input_size(); ++i) {
+            ss << brgemm->get_input_element_type(i);
+            if (i != brgemm->get_input_size() - 1) {
+                ss << ";";
+            }
+        }
+        ss << ",";
+        for (size_t i = 0; i < brgemm->get_output_size(); ++i) {
+            ss << brgemm->get_output_element_type(i);
+            if (i != brgemm->get_output_size() - 1) {
+                ss << ";";
+            }
+        }
+        ss << ",";
+        for (const auto& input : brgemm->inputs()) {
+            const auto& shape = input.get_partial_shape();
+            if (shape.is_dynamic()) {
+                ss << "?";
+            } else {
+                for (const auto& dim : shape) {
+                    ss << static_cast<int64_t>(dim.get_length()) << " ";
+                }
+                ss.seekp(-1, ss.cur);
+            }
+            ss << ";";
+        }
+        ss.seekp(-1, ss.cur);
+        ss << ",";
+        for (const auto& output : brgemm->outputs()) {
+            const auto& shape = output.get_partial_shape();
+            if (shape.is_dynamic()) {
+                ss << "?";
+            } else {
+                for (const auto& dim : shape) {
+                    ss << static_cast<int64_t>(dim.get_length()) << " ";
+                }
+                ss.seekp(-1, ss.cur);
+            }
+            ss << ";";
+        }
+        ss.seekp(-1, ss.cur);
+        ss << ",";
+        int64_t M, N, K;
+        std::tie(M, N, K) = get_brgemm_dimensions(brgemm_expr);
+        ss << M << "," << N << "," << K << ",";
+        int64_t m_block, n_block, k_block;
+        std::tie(m_block, n_block, k_block) = get_blocking_params(brgemm_expr);
```

Comment:
We don't need to call `get_blocking_params` here: the blocking params were already set to descriptors. Please extract the blocking params from descriptor's subtensors: this can be done by analogy with `get_brgemm_dimensions`

Reply:
Thanks, addressed